mmfunctions

```diff
@@ -12,6 +12,15 @@ elif [ ! "${CONFIG}" = "Y" -a "${REQUIRECONFIG}" = "Y" ] ; then
     exit 1
 fi
 
+#Setup differences for systems
+if [[ "$(uname -s)" = "Darwin" ]] ; then
+    FFMPEGFONTPATH='/Library/Fonts/Andale Mono.ttf'
+    XMLSTARLET='xml'
+else
```

Comment:
I suggest to use an explicit
```
elif [[ $(uname -s)" = "Linux" ]]
```
and something like
```
else
    echo "Error: Unsupported OS. Exiting."
    exit 1
fi
```

Reply:
That makes sense to me!

I just noticed one thing, on linux in windows `uname -s` returns `Linux` which I assume would cause this to still assign a bad file path for the text file in that environment.

@retokromer, I know you have been running more of these scripts in the linux on windows environment, have you run into any problems due to the lack of specificity in using `uname`?

Reply:
I cannot remember a specific problem, but I will check as soon as I am back at the lab.